### ISSUE TITLE
Incomplete title when 3D is in title

### ISSUE THREAD

**Issue by h3llrais3r** [guessit-io/guessit#588]
```
(test) C:\Tools\Python27-virtual\test\Scripts>guessit --version
+-------------------------------------------------------+
+                   GuessIt 3.0.3                       +
+-------------------------------------------------------+
+                   Rebulk 1.0.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+

(test) C:\Tools\Python27-virtual\test\Scripts>guessit "A Very Harold & Kumar 3D Christmas (2011).mkv"
For: A Very Harold & Kumar 3D Christmas (2011).mkv
GuessIt found: {
    "title": "A Very Harold & Kumar",
    "other": "3D",
    "release_group": "Christmas",
    "year": 2011,
    "container": "mkv",
    "type": "movie"
}
```

**Comment by ratoaq2** [comment]
Another tricky one

**Comment by h3llrais3r** [comment]
Indeed a tricky one :smile: 
For now, I use custom options as workaround.
```
custom_options = dict(options) if options else dict()
    custom_options.update(dict(expected_title=expected_titles, expected_group=expected_release_groups))
    result = default_api.guessit(string, options=custom_options)
```

**Comment by ratoaq2** [comment]
Fixed by a6f234f404c639a4fde74d37f63def7b27e2b01e
